randovania/data/json_data/prime1.txt

```diff
@@ -1580,411 +1583,799 @@ Tallon Overworld
+  > Dock to Main Ventilation Shaft Section A
+      Wave Beam and Thermal Visor
 
 ----------------
 Transport to Chozo Ruins East
 Asset id: 2318493278
-> Dock to Transport Tunnel C; Heals? False
-  * Normal Door to Transport Tunnel C/Dock to Transport to Chozo Ruins East
+> Dock to Transport Tunnel C; Heals? False; Spawn Point
+  * Ice Door to Transport Tunnel C/Dock to Transport to Chozo Ruins East
+  > Elevator to Chozo Ruins - Transport to Tallon Overworld East
+      Scan Visor
+
+> Elevator to Chozo Ruins - Transport to Tallon Overworld East; Heals? False
+  * Teleporter to Chozo Ruins/Transport to Tallon Overworld East
+  > Dock to Transport Tunnel C
+      Trivial
 
 ----------------
 Transport to Magmoor Caverns East
 Asset id: 366411659
-> Dock to Transport Tunnel B; Heals? False
+> Dock to Transport Tunnel B; Heals? False; Spawn Point
   * Normal Door to Transport Tunnel B/Dock to Transport to Magmoor Caverns East
+  > Elevator to Magmoor Caverns - Transport to Tallon Overworld West
+      Scan Visor
+
+> Elevator to Magmoor Caverns - Transport to Tallon Overworld West; Heals? False
+  * Teleporter to Magmoor Caverns/Transport to Tallon Overworld West
+  > Dock to Transport Tunnel B
+      Trivial
 
 ----------------
 Main Ventilation Shaft Section A
 Asset id: 1724639367
-> Dock to Main Ventilation Shaft Section B; Heals? False
+> Dock to Main Ventilation Shaft Section B; Heals? False; Spawn Point
   * Normal Door to Main Ventilation Shaft Section B/Dock to Main Ventilation Shaft Section A
+  > Dock to Reactor Core
+      Trivial
 
 > Dock to Reactor Core; Heals? False
   * Normal Door to Reactor Core/Dock to Main Ventilation Shaft Section A
+  > Dock to Main Ventilation Shaft Section B
+      Trivial
 
 ----------------
 Reactor Core
 Asset id: 3993592474
-> Dock to Reactor Access; Heals? False
+> Dock to Reactor Access; Heals? False; Spawn Point
   * Normal Door to Reactor Access/Dock to Reactor Core
+  > Dock to Main Ventilation Shaft Section A
+      Space Jump Boots
 
 > Dock to Main Ventilation Shaft Section A; Heals? False
   * Normal Door to Main Ventilation Shaft Section A/Dock to Reactor Core
+  > Dock to Reactor Access
+      Wave Beam and Thermal Visor
 
 ----------------
 Reactor Access
 Asset id: 4215436672
-> Dock to Cargo Freight Lift to Deck Gamma; Heals? False
+> Dock to Cargo Freight Lift to Deck Gamma; Heals? False; Spawn Point
   * Normal Door to Cargo Freight Lift to Deck Gamma/Dock to Reactor Access
+  > Dock to Reactor Core
+      Trivial
+  > Dock to Savestation
+      Trivial
 
 > Dock to Reactor Core; Heals? False
   * Normal Door to Reactor Core/Dock to Reactor Access
+  > Dock to Cargo Freight Lift to Deck Gamma
+      Wave Beam and Thermal Visor
+  > Dock to Savestation
+      Trivial
 
 > Dock to Savestation; Heals? False
   * Normal Door to Savestation/Dock to Reactor Access
+  > Dock to Cargo Freight Lift to Deck Gamma
+      Wave Beam and Thermal Visor
```

Comment:
Requirement not needed, as this path is accounted for from the Reactor Core Door